app/views/partials/code.njk

```diff
@@ -3,6 +3,12 @@
 <pre><code class="govuk-!-f-16">{{- componentHtml | e -}}</code></pre>
 
 <h4 class="govuk-heading-s">Macro</h4>
-<pre class="govuk-!-mb-r0 govuk-!-f-16"><code>{% raw %}{{ {% endraw %}{{ componentName | componentNameToMacroName }}(
-  {{- item.data | dump(2) -}}
-  ){% raw %} }}{% endraw %}</code></pre>
+<pre class="govuk-!-mb-r0 govuk-!-f-16">
```

Comment:
I think the new line here has changed the way the markdown is being generated, as all of the READMEs now include this div, rather than using the usual markdown indentation to signify a code block.